### PR TITLE
Add section about mdns discoverable KMS

### PR DESCRIPTION
**Pull request by jimmykarily**
Part of: https://github.com/kairos-io/kairos/issues/2087

### CODE REVIEW

**Review by mauromorales — COMMENTED** [review]
lgtm but what about all the rest of the docs you were showing us today, I think that example would be fantastic to have

**Comment by jimmykarily** [comment]
> lgtm but what about all the rest of the docs you were showing us today, I think that example would be fantastic to have

Yes I know. I felt the example was too "local" so I just [linked to that document](https://github.com/kairos-io/kairos-docs/pull/134/files#diff-f0ad8542aad47e509822a574da710b392abb32a14fc4aeeefc62cfbb82d25d9eR314). Maybe I should just move the whole example here? I feel it's a bit too much for production docs.

**Comment by jimmykarily** [comment]
> > lgtm but what about all the rest of the docs you were showing us today, I think that example would be fantastic to have
> 
> Yes I know. I felt the example was too "local" so I just [linked to that document](https://github.com/kairos-io/kairos-docs/pull/134/files#diff-f0ad8542aad47e509822a574da710b392abb32a14fc4aeeefc62cfbb82d25d9eR314). Maybe I should just move the whole example here? I feel it's a bit too much for production docs.

Or I could just move the whole document in the docs repo and link to it with the title "Instructions for a local test of discoverable kms" or something.

File: content/en/docs/Advanced/partition_encryption.md [hunk 1]
```diff
@@ -69,6 +71,7 @@ kcrypt:
 | `kcrypt.challenger.nv_index` | Custom Non-Volatile index to use to store encoded blobs |
 | `kcrypt.challenger.c_index` | Custom Index for the RSA Key pair |
 | `kcrypt.challenger.tpm_device` | Custom TPM device |
+| `kcrypt.challenger.mdns` | Discover KMS using mdns |
```
Comment (mudler) [inline content/en/docs/Advanced/partition_encryption.md:74]: is this a boolean?

Comment (mudler) [inline content/en/docs/Advanced/partition_encryption.md:74]:
```suggestion
| `kcrypt.challenger.mdns` | Discover KMS using mdns. Defaults to false |
```

Comment (jimmykarily) [reply content/en/docs/Advanced/partition_encryption.md:74]: yes

**Review by mauromorales — APPROVED** [review]
if we are linking to it that should be good enough

**Review by mudler — APPROVED** [review]

**Pull request merged**